doc/howtos/vulnerability-management.rst

```diff
@@ -0,0 +1,145 @@
+Ostro OS vulnerability handling
+###############################
+
+Introduction
+============
+
+Software vulnerabilities are found continuously even in mature pieces
+of software. This means that one of the key objectives of a Linux
+distribution is pushing fixes to vulnerabilities down to the end users.
+Ostro OS images will not be used as-is by end users, but Ostro OS
+will be the base software distribution for many products the Ostro OS
+customers are developing. These products must be kept secure for their
+expected lifetime. The published Ostro OS images are for development
+purposes only and should not be used in real products.
+
+Finding vulnerabilities
+=======================
+
+The Ostro Project security team must find out about vulnerabilities in
+Ostro OS. These can be new ones or introduced when Ostro OS content is
+changed. The `cve-check-tool <https://github.com/ikeydoherty/cve-check-
+tool>`_ is run during Ostro OS builds. The tool creates a file that
+maps current Ostro OS recipes against known vulnerabilities for the
+particular software versions. This file provides information about open
+CVE issues affecting components in Ostro OS, but has some limitations
+because not all CVE issues are exactly mapped to packages. For those
+components, such as Linux kernel, CVEs and vulnerability reports have
+to be investigated separately. Also, cve-check-tool doesn't report
+vulnerabilities that are not found in the vulnerability database.
+
+When Ostro OS -specific vulnerabilities are found that don't have a CVE
+entry, the person finding them can notify the Ostro OS Security Team by
+reporting the vulnerability using the secure contact email address and
+public key for communication encryption.
+
+Ostro project has a Security Officer (with a backup person) who reads
+the email address, follows the report generated by ``cve-check-tool``,
+follows the security advisories provided by the Yocto Project in yocto-
+security@yoctoproject.org mailing list, and follows the relevant open-
+source component vulnerability reporting mailing lists such as oss-
+security. The person is responsible for following up with vulnerability
+reports. The correct procedure is to analyze the issue and then file a
+report to Ostro OS issue tracker if required.
+
+An important consideration is that Ostro Project does not catch the
+vulnerabilities in third-party software that our customers might be
+using. These customers need to track the potential security
+vulnerabilities in those software components themselves or rely on the
+Yocto project or other upstream meta-layer maintainers to handle the
+detection and fixing of the vulnerabilities.
+
```

Comment:
I don't think "third-party" is clear enough here, since readers of this document may be consumers who consider Ostro OS to be third party code within their models.  Maybe "software not part of the core Ostro OS but deployed by customers" shortened to "non-core software" ?